Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -268,16 +269,19 @@ private void PrintTaskRequested(PrintManager sender, PrintTaskRequestedEventArgs
                             case PrintTaskCompletion.Failed:
                             {
                                 OnPrintFailed?.Invoke();
+                                _canvasContainer.RequestedTheme = ElementTheme.Default;
```

Comment:
Why not do this once outside of switch statement?

Is there a reason we have to keep resetting this property?